mabwiser/mab.py

```diff
@@ -107,15 +105,14 @@ class LinGreedy(NamedTuple):
         """
         epsilon: Num = 0.1
         l2_lambda: Num = 1.0
-        arm_to_scaler: Dict[Arm, Callable] = None
+        scale: bool = False
```

Comment:
Why default to `False`? To stay somewhat consistent with current behavior?

Reply:
Yes mainly.